src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -5,6 +5,7 @@ import Head from 'next/head';
 import CollectionGallery from './CollectionGallery';
 import useBackButton from 'hooks/useBackButton';
 import ActionText from 'components/core/ActionText/ActionText';
+import { baseUrl } from 'utils/baseUrl';
```

Comment:
nit: unused import?

Reply:
might consider wiring up CI and ensuring lint passes on the PR before merging so ya'll don't have to be the human linters!